pkgs/misc/cups/drivers/mfcl2740dwlpr/default.nix

```diff
@@ -0,0 +1,45 @@
+{ pkgs, stdenv, fetchurl, dpkg, makeWrapper, coreutils, ghostscript, gnugrep, gnused, which, perl }:
```

Comment:
You can take the `pkgsi686Linux` argument here directly to not have to use `pkgs`